<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>AES(cpuid to serial number)</string>
    </property>
    <widget class="QCheckBox" name="cbExpire">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Expire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leCPUID">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>18</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dteExpire">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>18</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>CPUID</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteSerialNumberGen">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Number:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbGenerateSn">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>170</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>to Serialnumber</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCpuidDt">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>170</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>to CPUID &amp;&amp; datetime</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>RSA(serial number to authorized code)</string>
    </property>
    <widget class="QPushButton" name="pbGenerate">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Encrypt to AuthCode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Number:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteSerialNumberIn">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>40</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>114</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Authorisation Code:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteAuthCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCreateRSAKeys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create RSA Keys</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbVerify">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt to SN.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
